src/services/commentService.js

```diff
@@ -0,0 +1,68 @@
+import models from '../database/models';
+
+const { Users, Trips, Comments } = models;
+
+const checkCommentOwner = async (commentId, userId) =>{
+  try{
+    const user = await Users.findOne({ where: { id: `${userId}` } });
+    const comment = await Comments.findOne({where:{id: commentId}});
+    const isOwner = comment.UserId === userId;
+    return user.hasComment(comment) && isOwner
+  }
+ catch(error){
+   return {error: error.message}
+ }
+} 
+
+const createComment = async (userId, tripId, aComment) => {
+  try {
+    const user = await Users.findOne({ where: { id: `${userId}` } });
+    const trip = await Trips.findOne({ where: { id: `${tripId}` } });
+    const comment = await Comments.create({
+      UserId: user.id,
+      TripId: trip.id,
+      comment: aComment
+    });
+    return comment;
+  } catch (error) {
+    return { error: error.message };
+  }
+};
+
+const fetchComments = async (tripId) => {
+  const getRecord = await Comments.findAll({
+    where: { TripId: `${tripId}` },
+    include: [
+      {
+        model: models.Users,
+        attributes: ['id', 'email', 'firstname', 'lastname']
+      },
+      {
+        model: models.Trips,
+        attributes: ['id', 'userId', 'accomodationId', 'destination']
```

Comment:
Remove userId, accomodationId from the response since you have fetched the user via association